File: pkg/pwrgrep/testdata/fixtures/csharp/Variables.cs
public class Variables
{
    // ruleid: cs-variables
    private const int Limit = 10;
    // ruleid: cs-variables
    private string name;

    void F()
    {
        // ruleid: cs-variables
        var list = new List<string>();
        // ruleid: cs-variables
        int x = 1;
        // ok: cs-variables
        x = 2;
    }
}

File: pkg/pwrgrep/testdata/fixtures/python/variables.py
# ruleid: python-variables
LIMIT = 10


def f(self, d):
    # ruleid: python-variables
    x = 1
    # ruleid: python-variables
    a, b = 1, 2
    # ruleid: python-variables
    name: str = "x"
    # ok: python-variables
    self.port = x
    # ok: python-variables
    d["k"] = a
    # ok: python-variables
    x == b
